packages/perspective/src/js/perspective.js

```diff
@@ -797,37 +797,21 @@ module.exports = function(Module) {
     };
 
     /**
-     * Expand the tree down to `depth`.
+     * Set expansion `depth` pf the pivot tree.
      *
      */
-    view.prototype.expand_to_depth = async function(depth) {
+    view.prototype.set_depth = async function(depth) {
```

Comment:
Given that this is a breaking change would it not be better to deprecate the old functions? (Unless they’ve not been released from master yet?)